src/current/v23.2/differences-in-metrics-between-third-party-monitoring-integrations-and-db-console.md

```diff
@@ -0,0 +1,50 @@
+## CockroachDB’s Timeseries Database
+
+CockroachDB stores metrics data in its own internal timeseries database (TSDB). While CockroachDB exposes [point-in-time metrics]({% link {{ page.version.version }}/monitor-cockroachdb-with-prometheus.md %}) via its [Prometheus endpoint]({% link {{ page.version.version }}/monitoring-and-alerting.md %}#prometheus-endpoint),  `/_status/vars`, it also periodically scrapes and writes this information to its own timeseries database storage. This data is scraped every 10 seconds and stored at that resolution. After a period of time determined by the [`timeseries.storage.resolution_10s.ttl` cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}#setting-timeseries-storage-resolution-10s-ttl), that 10 second resolution data is compacted into a 30 minute resolution. After another period of time determined by the [`timeseries.storage.resolution_30m.ttl cluster setting`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-timeseries-storage-resolution-30m-ttl), the 30 minute resolution data is deleted.
+
+The data in TSDB is used to populate metrics charts in DB Console. TSDB provides its own implementations to compute functions, such as rate-of-change, maximum, sum, etc. It also provides its own implementation to perform downsampling.
+
+## Different implementations 
+
+Systems, such as Datadog, Amazon CloudWatch, and Prometheus/Grafana, have their own flavor of timeseries database technology. The way they store data on disk, the query languages they expose to explore the data, and their implementations of downsampling and functions, such as rate-of-change, may differ from one another and from TSDB. 
+
+## Different resolutions
+
+Depending on the time range selected, the resolution in DB Console will be either 10-second or 30-minute. The data that is exported to third-party systems may be provided at different resolutions. For example, you might have a 10-second resolution in Prometheus, but the same metric in Amazon CloudWatch might have a 1-minute resolution. The resolution in Datadog can differ as well.
+
+Different systems have different polling frequencies. Since TSDB is fairly high fidelity, polling and storing values every 10 seconds, it can capture small changes. A third-party system, such as Datadog, may poll less frequently so it is very possible for it to not capture a small change. 
+
+In the following example, given the timeline and values for unavailable ranges, the count for the unavailable ranges metric briefly was non-zero at times `t3` and `t4` as scraped by TSDB. Since Datadog polled for data at times `t0` and `t6`, it did not capture the sudden unavailable range burst because the change was not long enough for the times that Datadog made its scrape. 
+
+time | t0 | t1 | t2 | t3 | t4 | t5 | t6
+-----|----|----|----|----|----|----|----
+TSDB scrapes every 10s | 0 | 0 | 0 | 5 | 4 | 0 | 0
+Datadog scrapes every 60s | 0 | - | - | - | - | - | 0
+
+## Different results
```

Comment:
nit: Can we axe this "Different results" section? I think it might create confusion and might prompt readers to try to pick apart the example and come to us with more questions. 

I think the table in "Different resolutions" is probably sufficient enough as an example.